Apps/Examples/Examples/All Examples/Annotations/CircleAnnotationExample.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import MapboxMaps
+
+@objc(CircleAnnotationExample)
+final class CircleAnnotationExample: UIViewController, ExampleProtocol {
```

Comment:
On Android this example allows to drag circle annotation, looks like there is no SDK counterpart on iOS. Is it possible to enable drag with lower level API?

Reply:
No, we haven't built anything to support this yet. Developers would have to build their own solution.